tla-pp/src/test/scala/at/forsyte/apalache/tla/pp/TestNormalizer.scala

```diff
@@ -63,4 +87,32 @@ class TestNormalizer extends FunSuite with BeforeAndAfterEach {
         .typed(types, "b")
     assert(expected == output)
   }
+
+  test("""x => y ~~> ~x \/ y""") {}
+
+  test("""~ (x => y) ~~> x /\ ~y""") {}
+
+  test("""(~FALSE <=> x) ~~> TRUE <=> x""") {}
+
+  test("""~ (~FALSE <=> x) ~~> TRUE \neq x""") {}
```

Comment:
Why do we preserve `<=>` in one case and produce `\neq` in another?

Reply:
I believe I've made a confusion because I didn't notice that `TlaBoolOper.equiv` and `TlaOper.eq` are two different things. However, I still don't think I understand this scenario of normalization. Why are we transforming `equiv` to `eq` or `neq`?  Are they different operators before this normalization and the same operator after?
```scala
    case equiv @ OperEx(TlaBoolOper.equiv, left, right) =>
      val nnfNonNegated = nnf(neg = false)
      // we do not negate the arguments but recurse to deal with the negations below the tree
      if (!neg) {
        OperEx(TlaOper.eq, nnfNonNegated(left), nnfNonNegated(right))(equiv.typeTag)
      } else {
        OperEx(TlaOper.ne, nnfNonNegated(left), nnfNonNegated(right))(equiv.typeTag)
      }
```
https://github.com/informalsystems/apalache/blob/normalizer-tests/tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/Normalizer.scala#L70

Reply:
`<=>` is literally just the `Bool`-typed variant of `=`. Since we support `=` in its full generality, rewriting `<=>` to `=` shrinks the language subset that needs top be supported.

Reply:
Right. We can see `<=>` as a convenient form of `=` on Booleans. Since normalization happens after type checking, it does not really matter that we use `=` instead of `<=>`. Also, I was out of context here. Negating `A <=> B` into `A /= B` is the most optimal solution. So let's use `=` and `/=`.